### ISSUE TITLE
No push notification for new order

### ISSUE THREAD

**Issue by reginabally** [woocommerce/woocommerce-ios#10148]
**Describe the bug**
Multiple reports of no push notification when a new order was placed in the store.

**To Reproduce**
1. Visit your test store and place an order
2. Observe that no push notification is received on the device

Notes:

1. Jetpack Sync doesn't seem to be an issue because the order can be found using MC Site Explorer tool.
2. The app prompts the notification if we send a test push notification via the MC Push Notifications tool.

**Expected behavior**
Push notification is sent when a new order is placed.

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone 14
- iOS version: 16.5.1
- WooCommerce iOS version: 14.1.0.3

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://woo20210709.mystagingwebsite.com
Site address (URL): https://woo20210709.mystagingwebsite.com
WC Version: 7.8.1
REST API Version: ✔ 7.8.1
WC Blocks Version: ✔ 10.2.4
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 7.4.33
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 8.0.1
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.4.25-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.8.1
WC Database Prefix: wp_
Total Database Size: 9.92MB
Database Data Size: 6.43MB
Database Index Size: 3.49MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.06MB + Index: 0.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.05MB + Index: 0.09MB + Engine InnoDB
wp_itsec_bans: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_itsec_dashboard_events: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_itsec_distributed_storage: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_itsec_fingerprints: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_itsec_geolocation_cache: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_itsec_lockouts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_itsec_logs: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_itsec_mutexes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_itsec_opaque_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_itsec_temp: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_itsec_user_groups: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_mapping_to_external_entities: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_scheduled_tasks: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_bounces: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscribers: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_user_agents: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mo2f_network_blocked_ips: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mo2f_network_email_sent_audit: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mo2f_network_transactions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mo2f_network_whitelisted_ips: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mo2f_user_details: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mo2f_user_login_info: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 1.06MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 0.27MB + Index: 0.22MB + Engine InnoDB
wp_posts: Data: 0.11MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.08MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cardsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wpns_attack_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpns_backup_report: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpns_files_scan: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpns_ip_rate_details: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpns_malware_hash_file: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wpns_malware_scan_report: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpns_malware_scan_report_details: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpns_malware_skip_files: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wp_phpmyadmin_extension__errors_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_indexable: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 38
gift_card: 1
mailpoet_page: 1
page: 8
post: 4
product: 44
product_variation: 51
revision: 8
shop_coupon: 3
shop_order: 61
shop_order_refund: 3
wp_global_styles: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (2) ###

Jetpack: by Automattic – 12.3-beta
WooCommerce: by Automattic – 7.8.1

### Inactive Plugins (18) ###

Akismet Anti-Spam: Spam Protection: by Automattic - Anti Spam Team – 5.2
Food Store - Online Food Delivery & Pickup: by WP Scripts – 1.4.7.3
iThemes Security: by iThemes – 8.1.6
MailPoet: by MailPoet – 4.20.0
miniOrange 2 Factor Authentication: by miniOrange – 5.7.1
Payment Plugins for Stripe WooCommerce: by Payment Plugins
support@paymentplugins.com – 3.3.43

Variation Swatches for WooCommerce: by Emran Ahmed – 2.0.23
WCS staging: by mb – 1.0
wePOS - Point Of Sale (POS) for WooCommerce: by weDevs – 1.2.6
WooCommerce Admin: by WooCommerce – 3.3.2
WooCommerce Gift Cards: by WooCommerce – 1.16.1
WooCommerce Payments: by Automattic – 6.1.1
WooCommerce Shipping & Tax: by WooCommerce – 2.2.5
WooCommerce Smart Coupons: by StoreApps – 5.3.0
WooCommerce Stripe Gateway: by WooCommerce – 7.4.1
WooCommerce Subscriptions: by WooCommerce – 4.2.0
YITH WooCommerce Gift Cards: by YITH – 4.1.0
Yoast SEO: by Team Yoast – 20.10

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
gift-card (gift-card)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: –
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Shop base: #5 - /
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.3.0
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: core-profiler
minified-js
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-07-03 00:52:56 +00:00
Options: ✔
Notes: 163
Onboarding: skipped

### Action Scheduler ###

Complete: 56
Oldest: 2023-06-02 09:42:45 +0000
Newest: 2023-07-02 02:15:09 +0000

Failed: 37
Oldest: 2023-02-15 01:22:54 +0000
Newest: 2023-06-02 09:42:55 +0000

Pending: 1
Oldest: 2023-07-03 00:44:46 +0000
Newest: 2023-07-03 00:44:46 +0000


### Status report information ###

Generated at: 2023-07-02 02:15:34 +00:00
````
</details>

**Comment by reginabally** [comment]
More reports:

* 6478868-zd-woothemes
* 6482769-zd-woothemes
* 6486257-zd-woothemes
* 6472821-zd-woothemes
* 6488207-zd-woothemes
* 6489863-zd-woothemes

**Comment by igotdes** [comment]
Also reported in #6492208-zen, which is about an AT site. I noticed that the WooCommerce Analytics Jetpack module is not enabled for this user. I've asked them to enable it in case it helps.

Strangely, the WooCommerce Analytics Jetpack module is also disabled on my own AT test sites. I'll check the other tickets too.

/Edit:

Here's what I found for the WooCommerce Analytics Jetpack module for the above tickets:

*   6478868-zd-woothemes > activated
*   6482769-zd-woothemes > deactivated
*   6486257-zd-woothemes > deactivated
*   6472821-zd-woothemes > activated
*   6488207-zd-woothemes > activated
*   6489863-zd-woothemes > deactivated

So, it's probably not related.

**Comment by joashrajin** [comment]
We have app logs in 6495610-zd-woothemes

**Comment by rachelmcr** [comment]
In case anyone else runs across this issue, this isn't caused by the app itself. Discussion and followup here (internal ref): peaMlT-82-p2

**Comment by jartes** [comment]
Another report from an atomic site here: 6499289-zd-woothemes

Suggested activating the WooCommerce Analytics Jetpack module in case it helps.

**Comment by reginabally** [comment]
@jartes Thanks for reporting this! Can you please create a follow-up ticket and send it to the mobile queue so we can follow up with them when the issue is fixed on Jetpack? Thanks!

**Comment by reginabally** [comment]
Marking this as closed since the issue has been resolved in Jetpack 12.3. Ref: peaMlT-82-p2#comment-317
